go/vt/vttablet/tabletserver/tabletenv/config.go

```diff
@@ -190,6 +190,7 @@ func registerTabletEnvFlags(fs *pflag.FlagSet) {
 	flagutil.DualFormatBoolVar(fs, &enableConsolidatorReplicas, "enable_consolidator_replicas", false, "This option enables the query consolidator only on replicas.")
 	fs.Int64Var(&currentConfig.ConsolidatorStreamQuerySize, "consolidator-stream-query-size", defaultConfig.ConsolidatorStreamQuerySize, "Configure the stream consolidator query size in bytes. Setting to 0 disables the stream consolidator.")
 	fs.Int64Var(&currentConfig.ConsolidatorStreamTotalSize, "consolidator-stream-total-size", defaultConfig.ConsolidatorStreamTotalSize, "Configure the stream consolidator total size in bytes. Setting to 0 disables the stream consolidator.")
+	fs.Int64Var(&currentConfig.ConsolidatorMaxQueryWait, "consolidator-max-query-wait", defaultConfig.ConsolidatorMaxQueryWait, "Configure the max wait size per query for consolidator.")
```

Comment:
```suggestion
	fs.Int64Var(&currentConfig.ConsolidatorQueryWaiterCap, "consolidator-query-waiter-cap", defaultConfig.ConsolidatorMaxQueryWait, "The maximum number of clients we allow to wait on the consolidator for each query.")
```

Reply:
The PR tile, description and the release notes will all need to be fixed along with the flag name/field change.